src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -250,6 +294,14 @@ class BasicSettings extends React.Component {
     }
   }
 
+  getTemplate () {
```

Comment:
I'm not sure this function is really needed since you're only calling it from `checkTimeZone()`.  Could just be inlined (like it is in `validateForm()` line 251) and be ok.